sdk/template/ci.yml

```diff
@@ -27,7 +27,7 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: template
-    TestPipeline: true
+    TestPipeline: false
```

Comment:
I assume you want to revert this before merging this change as you only did it for testing the non-test case.